src/consensus/pbft/pbftmetadata.h

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the Apache 2.0 License.
+#pragma once
+
+#include "node/entities.h"
+#include "node/rpc/jsonrpc.h"
+
+#include <msgpack-c/msgpack.hpp>
+#include <vector>
+
+namespace pbft
+{
+  struct Request
+  {
+    uint64_t actor;
+    uint64_t caller_id;
+    std::vector<uint8_t> caller_cert;
+    std::vector<uint8_t> raw;
+
+    MSGPACK_DEFINE(actor, caller_id, caller_cert, raw);
+  };
+
+  using PbftRequests = ccf::Store::Map<size_t, Request>;
```

Comment:
`uint64_t` for actor should be `ccf::ActorsType`.
`uint64_t` for caller_id should be `ccf::CallerId`.